charts/aws-efs-csi-driver/templates/controller-deployment.yaml

```diff
@@ -19,8 +19,8 @@ spec:
         app: efs-csi-controller
         app.kubernetes.io/name: {{ include "aws-efs-csi-driver.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
-      {{- if .Values.node.podAnnotations }}
-      annotations: {{ toYaml .Values.node.podAnnotations | nindent 8 }}
+      {{- if .Values.controller.podAnnotations }}
+      annotations: {{ toYaml .Values.controller.podAnnotations | nindent 8 }}
```

Comment:
`.Values.podAnnotations` was actually unused => not working (found out while using `kube2iam`)